app/scripts/metamask-controller.js

```diff
@@ -2676,20 +2669,21 @@ export default class MetamaskController extends EventEmitter {
     // subset of state for metamask inpage provider
     const publicConfigStore = new ObservableStore();
 
-    const selectPublicState = (chainId, { isUnlocked }) => {
+    const selectPublicState = async ({ isUnlocked }) => {
```

Comment:
The description of this method says:
```
Constructor helper: initialize a public config store.
This store is used to make some config info available to Dapps synchronously.
```
Can we still say this is made available synchronously?

Reply:
> This store is used to make some config info available to Dapps synchronously.

I believe this is referring to the state that the inpage provider exposes to the dapp in a synchronous manner